.circleci/config.yml

```diff
@@ -71,7 +71,7 @@ jobs:
       - run:
           name: Run cypress component tests
           command: |
-            TESTFILES=$(circleci tests glob cypress/component/**/*.ts | circleci tests split --split-by=timings --timings-type=filename | tr '\n' ',')
+            TESTFILES=$(circleci tests glob cypress/component/**/*.ts | circleci tests split --split-by=timings --timings-type=filename | awk '{if (NR>1) printf ","; printf "%s", $0} END {if (NR>0) printf " "}')
```

Comment:
This is a known issue with cypress, if you end a CSV with a comma the cypress command ignores it and runs everything, we are actually running all tests but now in 3 containers instead of splitting

Reply:
wonder if we have this issue in other projects too? (experience-packages, ..)

Reply:
experience-packages is not parallelised but yeah it's possible with the combo of test field splitting in circleci